pom.xml

```diff
@@ -81,6 +84,9 @@
     <!-- Bahmni Reports -->
     <bahmniReportsVersion>0.92.1</bahmniReportsVersion>
 
+    <!-- OpenERP Atomfeed Service -->
+    <openErpAtomfeedServiceVersion>0.93.1</openErpAtomfeedServiceVersion>
```

Comment:
Wondering how this was working previously.
Is this module really needed after all?